### PR TITLE
Yo Office Updates

### PR DESCRIPTION
**Pull request by nokafor**
Updated documentation to reflect changes to Yo Office flow. Changes will be released 6/22.

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-onenote-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-onenote-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-powerpoint-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-powerpoint-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-project-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-project-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-word-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201806221408081451-217%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=8RjnAdBW02KIsnkwlAF6%2BaNC%2B43tvTIjjlxjhmf4yHM%3D&st=2018-06-22T14%3A04%3A11Z&se=2018-07-23T14%3A09%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-onenote-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-onenote-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-powerpoint-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-powerpoint-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-project-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-project-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-word-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201806221410585345-217%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=cKpPIGkcInHRBJWlLmW5SPy4%2FpALArZi150Md7Dk6S0%3D&st=2018-06-22T14%3A07%3A17Z&se=2018-07-23T14%3A12%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/file-get-started-excel-angular.md [hunk 1]
```diff
@@ -39,14 +39,15 @@ An add-in's manifest file defines its settings and capabilities.
...
-    - **Would you like to create a new subfolder for your project?:** `No`
+
+    - **Choose a project type:** `Manifest`
     - **What do you want to name your add-in?:** `My Office Add-in`
     - **Which Office client application would you like to support?:** `Excel`
-    - **Would you like to create a new add-in?:** `No`
 
-    The generator will then ask you if you want to open **resource.html**. It isn't necessary to open it for this tutorial, but feel free to open it if you're curious! Choose yes or no to complete the wizard and allow the generator to do its work.
+
+    Once the wizard completes, a manifest file and resource file are available for you to build your project.
```
Comment (Lauragra) [inline docs/includes/file-get-started-excel-angular.md]: Global: Please change Once to After.

Comment (Lauragra) [inline docs/includes/file-get-started-excel-angular.md]: Global: Should this consistently be "After you complete the wizard"? The text is not currently consistent in all files.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-onenote-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-onenote-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-powerpoint-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-powerpoint-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-project-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-project-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-word-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201806221659106452-217%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gKUugJMHwM2QPFnxVEbDdFuTp9cMPIjr9jP8EL0RZno%3D&st=2018-06-22T16%3A55%3A11Z&se=2018-07-23T17%3A00%3A11Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/file-get-started-excel-angular.md (hunk 1 above)
Comment (nokafor) [reply docs/includes/file-get-started-excel-angular.md]: Yes they should be consistent. I just updated accordingly.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-onenote-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-onenote-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-powerpoint-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-powerpoint-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-project-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-project-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-word-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201806221659361163-217%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ShfKVD4bMyJtBZdEC77XAZxfUVsF6Xi4iDiyiwf8%2BsY%3D&st=2018-06-22T16%3A56%3A24Z&se=2018-07-23T17%3A01%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/file-get-started-excel-angular.md (hunk 1 above)
Comment (nokafor) [reply docs/includes/file-get-started-excel-angular.md]: Oh wait. Everywhere else says "Once you complete the wizard". So I updated these to match. Should we do it the other way and update everything to say "after"?

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-onenote-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-onenote-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-powerpoint-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-powerpoint-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-project-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-project-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-word-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201806221700168752-217%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qtoEyemDqG6jivfCPRw0bUVekX0gOJamTw5AyYvj%2BG0%3D&st=2018-06-22T16%3A57%3A30Z&se=2018-07-23T17%3A02%3A30Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/file-get-started-excel-angular.md (hunk 1 above)
Comment (Lauragra) [reply docs/includes/file-get-started-excel-angular.md]: Yes, sorry, it should be "After you complete the wizard". Thanks!

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-onenote-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-onenote-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-powerpoint-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-powerpoint-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-project-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-project-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-word-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201806221735313710-217%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=QeUCkvqAOseNutrNuhtCb3CH%2BNzU%2FWgWFAKhaP5fLMM%3D&st=2018-06-22T17%3A31%3A33Z&se=2018-07-23T17%3A36%3A33Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-onenote-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-onenote-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-powerpoint-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-powerpoint-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-project-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-project-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-word-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201806221736148609-217%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=0ju0LQFGLaSeB11jb2y5FVjiH%2FvpsxcNC6YKVBXDJ2M%3D&st=2018-06-22T17%3A32%3A44Z&se=2018-07-23T17%3A37%3A44Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/includes/file-get-started-excel-angular.md (hunk 1 above)
Comment (nokafor) [reply docs/includes/file-get-started-excel-angular.md]: Updated

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-onenote-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-onenote-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-powerpoint-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-powerpoint-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-project-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-project-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-word-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201806221736568583-217%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=OzlssxLKTO9BZMnmF5rADGN2SiftPqDiy5Nt%2Fpzf3Y0%3D&st=2018-06-22T17%3A33%3A48Z&se=2018-07-23T17%3A38%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/images/yo-office-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-onenote-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-onenote-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-powerpoint-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-powerpoint-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-project-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-project-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office-word-jquery.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office-word-jquery.png) | :white_check_mark:Succeeded | |
[docs/images/yo-office.png](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/images/yo-office.png) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-angular.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-angular.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-jquery.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-jquery.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-react.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-react.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-excel-vue.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-excel-vue.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-onenote.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-onenote.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-powerpoint.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-powerpoint.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-project.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-project.md) | :white_check_mark:Succeeded | |
[docs/includes/file-get-started-word.md](https://github.com/nokafor/office-js-docs-pr/blob/patch-1/docs/includes/file-get-started-word.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C6%5C22%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201806221737570328-217%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=WpQ%2FjyIWjvR%2BkZoucrQaD6rykxRo0tjndS3uZ5jrh8c%3D&st=2018-06-22T17%3A35%3A02Z&se=2018-07-23T17%3A40%3A02Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**